README.md

```diff
@@ -27,19 +27,16 @@ We've excluded the number emojis due to their lack of support on Chrome. We enco
 
 # Above and Beyond
 ## Running Gulp
```

Comment:
Also edited the README around running gulp to be a little friendlier and also fixed some grammar issues that were annoying me.